stupidchacha.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main()
+{
+	std::cout<<"Stupid chaminjun"<<std::endl;
+}
```

Comment:
You don't have return!! You are stupid!!!

Reply:
ok ok

Reply:
it's good!!